### PR TITLE
record login type

### PR DESCRIPTION
**Pull request by rybit**
People that are logging in via token refreshes should be noted - for active trackign later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -464,6 +464,24 @@ GoTrue exposes the following endpoints:
...
+## Testing
+
+Testing requires a few things.
+- docker installed on the machine for the mysql image
+- [pop](https://github.com/markbates/pop) and soda installed
+
+The necessary commands are:
+```
+go get github.com/markbates/pop/...
+go install github.com/markbates/pop/soda
```
Comment (calavera) [inline README.md]: these go get should probably be included in the `make deps` target, or we could have another target like `make dev_deps`.

Comment (rybit) [reply README.md]: hmmm actually those are there - but they didn't work 
```
 deps: ## Install dependencies.
   @go get -u github.com/markbates/pop/soda
   @go get -u github.com/golang/lint/golint
   @go get -u github.com/Masterminds/glide && glide install
```

Comment (rybit) [reply README.md]: removing for the time being

**Comment by rybit** [comment]
verified in staging:
password
> time="2018-02-14T19:50:24.776103111Z" level=info msg=Login action=login domain=identity-staging.services.netlify.com instance_id=c478dd2e-9e5f-4b0d-abc5-0dd3ace51fdb login_method=password metering=true user_id=9d7acce2-f38a-4072-bd58-ff324836e3f6

token 
> time="2018-02-14T21:17:47.907515772Z" level=info msg=Login action=login domain=identity-staging.services.netlify.com instance_id=c478dd2e-9e5f-4b0d-abc5-0dd3ace51fdb login_method=token metering=true user_id=9d7acce2-f38a-4072-bd58-ff324836e3f6

**Pull request merged**